modules/35_transport/edge_esm/sets.gms

```diff
@@ -153,10 +153,6 @@ CAMP_lscStrong
 
 EDGE_scenario(EDGE_scenario_all) "Selected EDGE-T scenario"
 
-*** sets for the reporting, to be consistent with *complex* realisation
-FE_Transp_fety35(all_enty) "FEs used in the transport module"  / fepet, fedie, feh2t, feelt, fegat/
-FE_Elec_fety35(all_enty)   "FE electricity sets (should be moved to core/sets asap)"  / feels, feelt /
```

Comment:
I guess you are probably changing the remind2 reporting to remove whatever code relies on these for the transport edge run. Just to double checking.

Reply:
I checked if deleted/renamed parameters/variables are used in our reporting